CMakeLists.txt

```diff
@@ -498,7 +498,7 @@ ADD_CUSTOM_TARGET(dist
 			COMMAND make clean
 			COMMAND rm -rf "${TMP}"
 			COMMAND mkdir -p "${TMP}"
-			COMMAND cp AUTHORS build_mingw32 build_mingw64 CMakeLists.txt configure COPYING INSTALL lmms.1 lmms.rc.in lmms.spec.in lmmsconfig.h.in lmmsversion.h.in README TODO "${TMP}"
```

Comment:
These are now moved into directories that are copied recursively, no need for explicitly copying them.